addons/60fps/addon.json

```diff
@@ -4,8 +4,13 @@
   "info": [
     {
       "type": "notice",
-      "text": "Most projects will not behave properly when running at 60FPS, because increasing the frame rate also increases the speed at which scripts run. This should only be used on projects that support 60FPS.",
+      "text": "Most projects will not behave properly when running at 60FPS, because increasing the framerate also increases the speed at which scripts run. This should only be used on projects that support 60FPS.",
```

Comment:
I would encourage you to review #6848 first. If you're not convinced with it, we may go ahead with removing this space, but I'd like to avoid sending this string for translation multiple times.